### PR TITLE
Nit: Fix in deviceplugin api description

### PR DESCRIPTION
**Pull request by eloyekunle**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**: Nit fix.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/apis/deviceplugin/v1beta1/api.proto [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// To regenerate api.pb.go run hack/update-device-plugin.sh
+// To regenerate api.pb.go run hack/update-generated-device-plugin.sh
```
Comment (dixudx) [inline pkg/kubelet/apis/deviceplugin/v1beta1/api.proto]: Oh, right. Somebody forgot to update this.
Good catch.

**Review by dixudx — APPROVED** [review]
/lgtm

**Comment by eloyekunle** [comment]
@dixudx @dchen1107 @mattjmcnaughton

**Review by mattjmcnaughton — APPROVED** [review]
thanks :)

/lgtm

**Comment by eloyekunle** [comment]
@mattjmcnaughton Any luck merging? :)

**Comment by mattjmcnaughton** [comment]
> @mattjmcnaughton Any luck merging? :)

I don't have the permissions to merge :) cc @dchen1107 

Although just a heads up, we are currently in code freeze for the 1.15 release, so we may need to wait until after code freeze lifts for this to actually be merged.

Thanks for your patience :)

**Comment by eloyekunle** [comment]
ping @dchen1107, any luck merging?

**Pull request closed without merging**